tests/installation/partitioning.pm

```diff
@@ -49,7 +49,9 @@ sub run {
         $cmd{resize}           = 'alt-r';
         $cmd{raw_volume}       = 'alt-r';
         $cmd{enable_snapshots} = 'alt-a';
-        $cmd{addpart}          = 'alt-r' unless is_tumbleweed;
+        $cmd{addpart}          = 'alt-r' if is_new_storage_ng;
```

Comment:
+1 to your solution. May I ask you to adjust solution @jknphy has applied [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning_raid.pm#L24) too, as you solve it. If not, feel free to leave it, I'll create separate PR.